UI/ChatGPT/ChatGPT/ChatGPT/appsettings.json

```diff
@@ -0,0 +1,9 @@
+{
+  "AppConfig": {
+    "Environment": "Production",
+    "ApiKey": "<insert api key here>"
```

Comment:
Which Api Key ? Should make it more obvious it's an ApiKey to access OpenAI services. Can we add comments there? We should provide the url to get one.

Reply:
Changes pushed in next commit.